src/clj/rems/api/applications.clj

```diff
@@ -143,12 +148,13 @@
   (let [is-handler? (or (contains? (set (applications/get-handlers (:application application))) user) ; old form
                         (contains? (get-in application [:application :possible-commands]) :see-everything))] ; dynamic
     (if is-handler?
-      application
+      (update-in application [:application] dissoc :invitation-tokens)
```

Comment:
couldn't this dissoc go outside the if?

Reply:
I think it does not make it any easier for the eyes like that if you try to combine the two cases. This is code about to be deleted or significantly refactored so I'd rather just leave it as it is for now.

Reply:
ok